src/pages/Thanks/ThanksPage.vue

```diff
@@ -289,6 +300,21 @@ export default {
 				);
 			}
 		}
+
+		// CORE-427 Thanks auto deposit upsell experiment
+		const autoDepositUpsellExp = this.apollo.readFragment({
+			id: 'Experiment:simple_thanks_share',
+			fragment: experimentVersionFragment,
+		}) || {};
+
+		this.simpleSocialShareVersion = autoDepositUpsellExp.version; // either 'a' or 'b' or undefined
+		if (this.autoDepositUpsellExpVersion) {
```

Comment:
This line and the line below in the `$kvTrackEvent` call should be `this.simpleSocialShareVersion` instead of `this.autoDepositUpsellExpVersion`

Reply:
done!